helm/ingress-controller/tests/credentials-secret_test.yaml

```diff
@@ -1,10 +1,35 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/quintush/helm-unittest/master/schema/helm-testsuite.json
```

Comment:
https://github.com/quintush/helm-unittest#test-suite-code-completion-and-validation I noticed this and it helped a bit while writing tests. it didn't seem easy to setup the other way in vscode